### DISCUSSION TITLE
Can't get rid of help screens

### DISCUSSION THREAD

**Discussion by floeby** [Leantime/leantime#507] [General]
Hi,

Running Leantime perfectly, but cannot get rid of help screens, each time you start Leantime they pop up.
Any ideas how to disable the funstion?

Kind regards,

Piet

**Comment by JulioSM98** [comment]
![image](https://user-images.githubusercontent.com/29544449/142258096-60bda350-3f4a-49d9-be78-37ba54c99907.png)
Clic in "Close and do not show this screen again" for each module

**Comment by marcelfolaron** [comment]
Do you see any errors in your error log? @JulioSM98 is right, clicking the "Close and do not show this screen again" link should close it for good.

**Comment by floeby** [comment]
Sorry for the late  reply...
clicking the "Close and do not show this screen again" makes the screen for e.g. reports or milestopnes go away for that session. 
However, as soon as you start a new session all the help screens are back.

I'm afraid I cannot find the error log....Could you tell me where I can find it?

kind regards,

Piet

**Reply by PinkieIcePie** [reply]
I have this same issue but my error.log is empty. 
Win server 2019, IIS.

**Comment by marcelfolaron** [comment]
The error log is in resources/logs/error.log

**Reply by floeby** [reply]
Found it:
got this error:
PHP Notice:  Undefined index: priority in public_html/src/domain/tickets/repositories/class.tickets.php on line 1126

Line 1126 is basically this:
            $stmn->bindValue(':priority', $values['priority'], PDO::PARAM_STR);
